src/evaluation.py

```diff
@@ -356,6 +360,23 @@ def conditionally_print_explanation(self):
         if self.should_explain_scoring:
             console.print(self.explanation_table, justify="center")
 
+    # Explanation Table to CSV
+    def conditionally_save_explanation_csv(self, evaluation_path):
+        if self.enable_evaluation_table_to_csv:
+            data = {col.header: col._cells for col in self.explanation_table.columns}
+
+            output_dir = os.path.join(
+                os.getcwd(),
+                f"{evaluation_path}.csv",
+            )
```

Comment:
let's use evaluation_output_dir here
```suggestion
    def conditionally_save_explanation_csv(self, file_path, evaluation_output_dir):
        if self.enable_evaluation_table_to_csv:
            data = {col.header: col._cells for col in self.explanation_table.columns}

            output_path = os.path.join(
               evaluation_output_dir,
                f"{file_path.stem}_evaluation.csv",
            )
```